apps/explorer/src/components/HomeMetrics/index.tsx

```diff
@@ -114,7 +114,7 @@ export function HomeMetrics() {
                     />
                     <FormattedStatsAmount
                         label="Addresses"
-                        tooltip="Total addresses counter"
+                        tooltip="Addresses that have received or sent atleast one transaction since network genesis"
```

Comment:
`atleast` -> `at least` 

Also something like `Addresses that have participated in at least one transaction since network genesis` is a little more clear to me